cfgrammar/src/lib/yacc/firsts.rs

```diff
@@ -201,8 +216,20 @@ mod test {
 
     #[test]
     fn test_first_no_subsequent_rules() {
+        let mut header = Header::new();
```

Comment:
It would be really nice to simplify a lot of this testsuite stuff with a macro.
I think I'll try to make a `test_support.rs` module, with a macro `header_for_yacckind!` or something to that effect.

Reply:
I did that in [a7aee25](https://github.com/softdevteam/grmtools/pull/547/commits/a7aee25a8140bb7d36e8376474c49267fd32bfc9).